.docker/readme.md

```diff
@@ -33,21 +31,28 @@ Follow these steps to setup a local WordCamp.org environment using [Docker](http
     git checkout 5.2
     ```
 
-1. Install the 3rd-party plugins, themes, and additional packages used on WordCamp.org. For this, you must have [Composer](https://getcomposer.org/doc/00-intro.md) installed. Once it is, change to the root directory of the project where the **composer.json** file is located.
+1. Install 3rd-party PHP packages used on WordCamp.org, including for development and testing. For this, you must have [Composer](https://getcomposer.org/doc/00-intro.md) installed. Once it is, change to the root directory of the project where the **composer.json** file is located.
```

Comment:
This threw me off, I wasn't sure if it meant the wp-content composer or root composer – we can assume the person going through this is in `mu` still, so why not say `cd ../..` in the bash instructions?

Also, the "including for development and testing" seems unnecessary, I think it could just be removed.

Reply:
Updated in e516dba7